Boop/Boop.xcodeproj/project.pbxproj

```diff
@@ -411,7 +409,7 @@
 			isa = PBXProject;
 			attributes = {
 				LastSwiftUpdateCheck = 1100;
-				LastUpgradeCheck = 1010;
+				LastUpgradeCheck = 1330;
```

Comment:
This and the rest of the file's edits I believe were modified by accepting XCode's suggestions. Was that ok?